data/transition-sites/beis.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: beis
+whitehall_slug: department-for-business-energy-and-industrial-strategy
+homepage: https://www.gov.uk/government/organisations/department-for-business-energy-and-industrial-strategy
+tna_timestamp: 20170612110308
+host: www.beis.gov.uk
+global: =410
```

Comment:
If we want a global redirect, this needs to be a `301` like this example https://github.com/alphagov/transition-config/blob/082688d330ea44c12e3688dc669ae6b1198b9a7a/data/transition-sites/auditcommission.yml#L7